run_unittests.py

```diff
@@ -181,5 +189,19 @@ def test_run_target_files_path(self):
         self.init(testdir)
         self.run_target('check_exists')
 
+    def test_qt5dependency_qmake_detection(self):
+        # Can't be sure that `qmake` is Qt5, so just try qmake-qt5.
+        if not shutil.which('qmake-qt5'):
+            raise unittest.SkipTest('qt5 not found')
+        # Disable pkg-config codepath and force searching with qmake/qmake-qt5
+        os.environ['PKG_CONFIG_LIBDIR'] = self.builddir
+        os.environ['PKG_CONFIG_PATH'] = self.builddir
```

Comment:
It doesn't leak because of this: https://github.com/mesonbuild/meson/blob/master/run_unittests.py#L65

We reset the environment after each test is torn down.